.travis.yml

```diff
@@ -0,0 +1,27 @@
+before_script:
+  - ruby --version
+
+env:
+  global:
+    - APM_TEST_PACKAGES="language-haml linter"
```

Comment:
Your last scenario _should_ be working... I'll take a look at this later tonight.